viewutil/viewutil.go

```diff
@@ -98,6 +98,7 @@ type BaseData struct {
 	Addr          string
 	Title         string // TODO: remove
 	Body          string // TODO: remove
+	ExtraData    []string
```

Comment:
Seems to fit other fields of `BaseData` well. What about a different name, such as `BodyAttributes`?